### DISCUSSION TITLE
GitHub pages site considered secure by some browsers and insecure by others

### DISCUSSION THREAD

**Discussion by reece** [community/community#40244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The static web site biocommons.com served by github pages. On my (ubuntu 22.04) laptop, Chrome and Firefox show that the certificate for biocommons.org is valid and that the site is secure. Chrome and Firefox on my Android 13 phone say that the connection is **not** secure.  

## Linux Chrome
![image](https://user-images.githubusercontent.com/109453/204197531-a140d8c7-fe4a-4bb8-98e0-7b7474f459f2.png)

## Linux Firefox
![image](https://user-images.githubusercontent.com/109453/204197508-013eb5f6-b4ca-4e9f-958e-ad5e891d73ce.png)

## Android Chrome
![image](https://user-images.githubusercontent.com/109453/204197707-aa9e1dad-1a4a-4689-b6cf-bf193bf04c47.png)

## Android Firefox
![image](https://user-images.githubusercontent.com/109453/204197732-358f689e-f824-4b72-895f-c7bbfdf500e0.png)

Safari also thinks that the site isn't secure?

Why is the certificate valid in some environments and not others?

**Comment by yoannchaudet** [comment]
If you are the owner of this Pages site you will want to go to the Pages settings and enable HTTPS redirection.

This is currently not enabled and the reason you don't always get the "Connection is secure" check. When you type `biocommons.org` in a navigation bar, some browser will assume HTTPS while some will not.

**Reply by reece** [reply]
That was it. Took immediate effect. Thank you.

For anyone with a similar issue, this is the option:
![image](https://user-images.githubusercontent.com/109453/205131845-66cbaae7-c3f1-405b-b86f-e812e33a84d4.png)
